resource/stack/stack.js

```diff
@@ -47,6 +47,14 @@ const getTemplate = Promise.coroutine(function* ( token, catalogOwner, catalog,
 	
 });
 
+function sendMsg(slack, msg) {
```

Comment:
This kind of error handling would be better served in the slack module itself. That way the behavior of sending a slack message in this service is consistent (i.e. fire and log errors)